monitoring/logging.sh

```diff
@@ -184,15 +185,15 @@ allMessage()
 #
 # log a warning message, execution continues
 warn()
-{ local DATETIME=`date +'%Y-%h-%d-T%H:%M:%S%z'`
+{ local DATETIME=`date +'%D %H:%M:%S'`
   MSG="[${DATETIME}] WARNING: $1"
   for warnlogfile in $SYSLOG $2 ; do
     if [[ -e $warnlogfile ]]; then
       local frame=0
-      while caller $frame; do
-         caller $frame >> $warnlogfile
-         ((frame++));
-      done
+ #     while caller $frame; do
```

Comment:
I find the execution tracing on warning messages to be very helpful ... is there a better way for us to be doing this?

Reply:
This was causing uncatchable chatter in the main window. I can look more into that call and see what options we have.

Reply:
I looked more into this. I understand now `caller` is to create basically a stack trace in the event of an error. I'll uncomment this, but we'll want to fix the underlying issue that is actually calling `warn`.

Reply:
That sounds great, I am open to better solutions.